ESSArch_Core/search/importers/tests/test_earderms.py

```diff
@@ -0,0 +1,1254 @@
+                    'format': 'sub format 1',
+                    'egenskaper': []
+                },
+                {
+                    'varde': 'sub value 2',
+                    'namn': 'sub name 2',
+                    'datatyp': 'sub data type 2',
+                    'format': 'sub format 2',
+                    'egenskaper': []
+                },
+            ],
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_egenskaper(root)
+
+        self.assertDictEqual(data, expected_data)
+
+
+class ParseEgetElementTests(TestCase):
+
+    def setUp(self):
+        self.importer = EardErmsImporter()
+
+    def test_parse_eget_element_with_one_level_of_element(self):
+        xml_string = '''
+            <root Namn='some name' DataTyp='some data type' Format='some format'>
+                <Varde>some value</Varde>
+            </root>
+        '''
+
+        expected_data = {
+            'varde': 'some value',
+            'namn': 'some name',
+            'datatyp': 'some data type',
+            'format': 'some format',
+            'element': [],
+            'egenskaper': [],
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_eget_element(root)
+
+        self.assertDictEqual(data, expected_data)
+
+    def test_parse_eget_element_with_no_Namn_attribute(self):
+        xml_string = '''
+            <root DataTyp='some data type' Format='some format'>
+                <Varde>some value</Varde>
+            </root>
+        '''
+
+        expected_data = {
+            'varde': 'some value',
+            'namn': None,
+            'datatyp': 'some data type',
+            'format': 'some format',
+            'element': [],
+            'egenskaper': [],
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_eget_element(root)
+
+        self.assertDictEqual(data, expected_data)
+
+    def test_parse_eget_element_with_egenskaper(self):
+        self.importer.parse_egenskaper = mock.Mock(return_value={'dummy_egenskap': 'dummy_value'})
+        xml_string = '''
+            <root Namn='some name' DataTyp='some data type' Format='some format'>
+                <Varde>some value</Varde>
+                <Egenskaper>
+                    <Egenskap>
+                    </Egenskap>
+                </Egenskaper>
+            </root>
+        '''
+
+        expected_data = {
+            'varde': 'some value',
+            'namn': 'some name',
+            'datatyp': 'some data type',
+            'format': 'some format',
+            'element': [],
+            'egenskaper': [{'dummy_egenskap': 'dummy_value'}],
+        }
+
+        root = get_xml_root_from_string(xml_string)
+        data = self.importer.parse_eget_element(root)
+
+        self.assertDictEqual(data, expected_data)
+
+    def test_parse_eget_element_with_name_prefix_should_remove_the_prefix(self):
+        xml_string = u'''
```

Comment:
The `u`-prefix is unnecessary in Python 3.3+

```suggestion
        xml_string = '''
```